spikeinterface/comparison/hybridunitsrecording.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Union
+import numpy as np
+from spikeinterface.core import BaseRecording, BaseSorting, NumpySorting
+from spikeinterface.comparison import AddTemplatesRecording
+from spikeinterface.extractors.toy_example import synthesize_random_firings
```

Comment:
I would move this synthesize_random_firings into a new core/generator.py module.
@alejoe91 ?

Reply:
agreed

Reply:
I couldn't find `core/generator.py` to move it there?

Reply:
you should make one :)

Reply:
Should I also move `core.testing_tools.generate_recording` to `core/generator.py`? (and same for `generate_sorting`)